modules/end-user-guide/partials/proc_creating-workspaces.adoc

```diff
@@ -2,38 +2,60 @@
 [id="creating-workspaces"]
 = Creating workspaces
 
-Since workspaces on the {orch-name} cluster are represented by custom resources, it is possible to create workspaces with {orch-name} APIs by applying custom resources to the cluster.
+Because workspaces on the {orch-name} cluster are represented by custom resources, and if your use case does not permit use of the {prod-short} dashboard, you can instead create workspaces with {orch-name} APIs by applying custom resources to the cluster.
```

Comment:
```suggestion
Because workspaces on the {orch-name} cluster are represented by custom resources, you can create workspaces with {orch-name} APIs by applying custom resources to the cluster instead of creating workspaces with the {prod-short} dashboard.
```

```suggestion
If your use case does not permit use of the {prod-short} dashboard, you can instead create workspaces with {orch-name} APIs by applying custom resources to the cluster. This is because {prod-short} workspaces are custom resources on the cluster level.
```

I find this sentence a bit awkward and thought of some suggestions, WDYT?

Reply:
Yes, you're right. Thank you for the suggestions.

Reply:
@dkwon17,  I've reworked your latter suggestion, WDYT?
```suggestion
If your use case does not permit use of the {prod-short} dashboard to create new workspaces, you can alternatively create workspaces with {orch-name} APIs by applying custom resources to the cluster.
```